### PR TITLE
feat: styling secondary 4 slice

### PR DESCRIPTION
**Pull request by nehasri89**

### CODE REVIEW

**Comment by tuncaulubilge** [comment]
One note: I'm working on the touch animations on the slices, and these tiles has a margin around it instead of a padding, which doesn't work well with the android ripple effect. I'll show you a sample

File: packages/slice-layout/src/templates/secondaryfour/styles.js [hunk 1]
```diff
@@ -4,19 +4,23 @@ const styles = {
...
-    padding: 5,
+    paddingBottom: 10,
     width: "50%"
   },
   itemContainer: {
     borderBottomWidth: 0.5,
     borderStyle: "solid",
     flexDirection: "row",
-    margin: 5
+    marginBottom: 12,
```
Comment (aronshamash) [inline packages/slice-layout/src/templates/secondaryfour/styles.js]: I'll been using `spacing(2)` for this but I know Zeplin says `12`. @tuncaulubilge  @jimhunty thoughts?

File: packages/slice-layout/src/templates/secondaryfour/styles.js [hunk 2]
```diff
@@ -4,19 +4,23 @@ const styles = {
...
+    marginLeft: 12,
+    marginRight: 12,
+    marginTop: 10
```
Comment (aronshamash) [inline packages/slice-layout/src/templates/secondaryfour/styles.js]: `spacing(2)`

File: packages/slice-layout/src/templates/secondaryfour/styles.js [hunk 3]
```diff
@@ -4,19 +4,23 @@ const styles = {
...
   },
   itemSeparator: {
-    borderLeftWidth: 0.25,
+    borderLeftWidth: 0.5,
     borderStyle: "solid",
-    margin: 5
+    marginLeft: 9,
```
Comment (aronshamash) [inline packages/slice-layout/src/templates/secondaryfour/styles.js]: Can use `marginHorizontal: spacing(2)`

**Comment by nehasri89** [comment]
Closing this as have done refactor in another PR

**Pull request closed without merging**